### PR TITLE
RDM-9587 FTs for the Conditional Post States

### PR DESCRIPTION
**Pull request by rafalkalita**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-9587

### Change description ###
FTs for the Conditional Post States

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/aat/resources/features/F-122 - Conditional Event Post States/F-122.feature [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    And the response [contains state: CaseAmended2, updated values for TextField, EmailField along with an HTTP-201 Created],
+
+#-----------------------------------------------------------------------------------------------------------------------
+  @S-122.7
+  Scenario: *(FieldA) will keep the state as is (End state: CaseCretated)
+
+    Given a user with [an active profile in CCD],
+    And a successful call [to create a token for case creation] as in [S-122-GetToken_CaseCreate],
+    And a successful call [to create a case] as in [FT_ConditionalPostState_Create_Case],
+    And a successful call [to get an event token for the case just created] as in [S-122-GetToken_UpdateCase],
```
Comment (kiran-yenigala-hmcts) [inline src/aat/resources/features/F-122 - Conditional Event Post States/F-122.feature:123]: "${[scenarioContext][parentContext][childContexts][S-122-GetToken_CaseCreate][testData][actualResponse][body][token]}"

**Review by kiran-yenigala-hmcts — COMMENTED** [review]
One comment otherwise looks good to me

File: src/aat/resources/features/F-122 - Conditional Event Post States/F-122.feature (hunk 1 above)
Comment (kiran-yenigala-hmcts) [reply src/aat/resources/features/F-122 - Conditional Event Post States/F-122.feature:123]: Sorry, Please ignore this, as this is a new update event, we need to generate the event_token again

Comment (kiran-yenigala-hmcts) [reply src/aat/resources/features/F-122 - Conditional Event Post States/F-122.feature:123]: Sorry, Please ignore this, as this is a new update event, we need to generate the event_token again

**Review by kiran-yenigala-hmcts — APPROVED** [review]

**Review by Olawale — APPROVED** [review]
LGTM

**Pull request merged**